src/explorer/SiteTreeItem.ts

```diff
@@ -143,17 +143,22 @@ export abstract class SiteTreeItem extends SiteTreeItemBase implements ISiteTree
         return !!(logsConfig.httpLogs && logsConfig.httpLogs.fileSystem && logsConfig.httpLogs.fileSystem.enabled);
     }
 
-    public async enableHttpLogs(): Promise<void> {
-        const logsConfig: WebSiteManagementModels.SiteLogsConfig = {
-            httpLogs: {
+    public async enableLogs(enableAppLogs: boolean): Promise<void> {
+        const logsConfig: WebSiteManagementModels.SiteLogsConfig = {};
+        if (enableAppLogs) {
```

Comment:
I don't get why one call to `enableLogs` checks the OS and one just passes `false`. 

Would it make sense to get rid of the flag and just check `!this.root.client.isLinux`?

Reply:
The instance with `false` is in the deployment path, I wasn't sure if we wanted to modify the behavior there. 
That's a good idea, it will shorten the code.